### PR TITLE
Dev 1

### PR DESCRIPTION
**Pull request by robertoratti**
Merging Fin3 dev and updating from upstream main

### CODE REVIEW

File: service/build.gradle.kts [hunk 1]
```diff
@@ -17,6 +20,58 @@ dependencies {
         it(Libraries.ScarletStreamAdapter)
         it(Libraries.ScarletWebsocket)
         it(Libraries.ScarletMessageAdapter)
+
+        // telemetry
+        it(Libraries.MicrometerPrometheusRegistry)
+
+        // ----- Misc -----
```
[the remark this answers is not in the document]
Comment (brazbro) [reply service/build.gradle.kts:27]: Thanks @Rathtrick I will remove other lines not in main branch.

File: extra/Omnibus.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
+const http = require("http");
```
[the remark this answers is not in the document]
Comment (brazbro) [reply extra/Omnibus.js:1]: @Rathtrick this was a test server I added to receive events from DCC sent to the bank's API. It is useful for suppressing connection errors in the log and seeing what DCC is sending.

**Review by brazbro — APPROVED** [review]
Updated dependencies, looks good.

**Review by Albert-fin3 — APPROVED** [review]
lgtm
Did not test
I was not involved in the original development/mods so i only took a superficial look...

**Pull request merged**